x-pack/plugins/siem/public/components/page/overview/overview_host_stats/index.tsx

```diff
@@ -6,75 +6,111 @@
-
 interface OverviewHostProps {
   data: OverviewHostData;
 }
 
 const overviewHostStats = (data: OverviewHostData) => [
   {
-    title:
+    description:
       has('auditbeatAuditd', data) && data.auditbeatAuditd !== null
         ? numeral(data.auditbeatAuditd).format('0,0')
         : getEmptyTagValue(),
-    description: i18n.AUDITBEAT_AUDITD,
+    title: (
+      <FormattedMessage
+        id="xpack.siem.overview.auditBeatAuditTitle"
+        defaultMessage="Auditbeat Audit"
+      />
+    ),
   },
   {
-    title:
+    description:
       has('auditbeatFIM', data) && data.auditbeatFIM !== null
         ? numeral(data.auditbeatFIM).format('0,0')
         : getEmptyTagValue(),
-    description: i18n.AUDITBEAT_FIM,
+    title: (
+      <FormattedMessage
+        id="xpack.siem.overview.auditBeatFimTitle"
+        defaultMessage="Auditbeat File Integrity Module"
+      />
+    ),
   },
   {
-    title:
+    description:
       has('auditbeatLogin', data) && data.auditbeatLogin !== null
         ? numeral(data.auditbeatLogin).format('0,0')
         : getEmptyTagValue(),
-    description: i18n.AUDITBEAT_LOGIN,
+    title: (
+      <FormattedMessage
+        id="xpack.siem.overview.auditBeatLoginTitle"
+        defaultMessage="Auditbeat Login"
+      />
+    ),
   },
   {
-    title:
+    description:
       has('auditbeatPackage', data) && data.auditbeatPackage !== null
         ? numeral(data.auditbeatPackage).format('0,0')
         : getEmptyTagValue(),
-    description: i18n.AUDITBEAT_PACKAGE,
+    title: (
+      <FormattedMessage
+        id="xpack.siem.overview.auditBeatPackageTitle"
+        defaultMessage="Auditbeat Package"
+      />
+    ),
   },
   {
-    title:
+    description:
       has('auditbeatProcess', data) && data.auditbeatProcess !== null
         ? numeral(data.auditbeatProcess).format('0,0')
         : getEmptyTagValue(),
-    description: i18n.AUDITBEAT_PROCESS,
+    title: (
+      <FormattedMessage
+        id="xpack.siem.overview.auditBeatProcessTitle"
+        defaultMessage="Auditbeat Process"
+      />
+    ),
   },
   {
-    title:
+    description:
       has('auditbeatUser', data) && data.auditbeatUser !== null
         ? numeral(data.auditbeatUser).format('0,0')
         : getEmptyTagValue(),
-    description: i18n.AUDITBEAT_USER,
+    title: (
+      <FormattedMessage
+        id="xpack.siem.overview.auditBeatUserTitle"
+        defaultMessage="Auditbeat User"
+      />
+    ),
   },
 ];
+
+export const DescriptionListDescription = styled(EuiDescriptionListDescription)`
+  text-align: right;
+`;
+
 export const OverviewHostStats = pure<OverviewHostProps>(({ data }) => (
-  <>
-    {overviewHostStats(data).map(item => (
-      <EuiStat
-        key={item.description}
-        textAlign="center"
-        title={item.title}
-        description={item.description}
-      />
+  <EuiDescriptionList type="column">
+    {overviewHostStats(data).map((item, index) => (
+      <React.Fragment key={index}>
```

Comment:
Previously the key used was `item.description`. Usually index is the last resort for a `key`. It will probably work here, but if you know that `title` or `description` will be unique it would serve as a better key.

Ref: https://reactjs.org/docs/lists-and-keys.html

Reply:
In this case, I used `index` because VSCode was reporting the below error when using `item.title` or `item.description`. Using `index` resolved the issue. Let me know there's a better way.

![image](https://user-images.githubusercontent.com/3884767/55647421-9e65b300-57ab-11e9-80c6-34d9e646eab6.png)

Reply:
I think since it returns `FormattedMessage` which is a JSX object now you really have to use `index` which will be fine as far as I can tell for this UI.